realm/realm-library/src/objectServer/java/io/realm/mongodb/auth/EmailPasswordAuth.java

```diff
@@ -164,6 +165,38 @@ public Void run() throws AppException {
         }.start();
     }
 
+    /**
+     * Calls the custom confirmation function on a user for a given email.
+     *
+     * @param email the email of the user.
+     * @throws AppException if the server failed to confirm the user.
+     */
+    public void callCustomConfirmationFunction(String email) throws AppException {
```

Comment:
We should make sure we call this method the same across SDK's. Right now it doesn't seem to be documented on the website (most likely because the old SDK did not have it). 

Especially since it is described as rerunning the registration. This makes me wonder if we should rename this to `rerunCustomConfirmationFunction` since it already ran once when registering and reword the javadoc appropriately?